<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>boughtCard</class>
 <widget class="QWidget" name="boughtCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>134</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#boughtCard{
border: 1px solid #000000; 
border-radius: 10px;
background-color: #ffffff;
padding: 10px;
}
</string>
  </property>
  <widget class="QLabel" name="iconLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="arrivalLabel">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#arrivalLabel{
font-size: 18px; 
font-weight: bold
}</string>
    </property>
    <property name="text">
     <string>到达时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>———————————————</string>
    </property>
   </widget>
   <widget class="QLabel" name="departimeLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#departimeLabel{
font-size: 18px; 
font-weight: bold
}</string>
    </property>
    <property name="text">
     <string>出发时间</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>日期</string>
    </property>
   </widget>
   <widget class="QLabel" name="deparcityLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>出发</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="gaipushButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#gaipushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 197, 82);
	border-radius: 10px;
	font-size: 16px;  
}
#gaipushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 163, 87);
	border-radius: 10px;
	
}
#gaipushButton:pressed {
    background-color: rgb(255, 163, 87);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px;  /* 鼠标按下时，字体缩小 */
}</string>
   </property>
   <property name="text">
    <string>改签</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tuipushButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#tuipushButton{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 197, 82);
	border-radius: 10px;
	font-size: 16px;  
}
#tuipushButton:hover{
	color: rgb(255, 255, 255);
	background-color: rgb(255, 163, 87);
	border-radius: 10px;
	
}
#tuipushButton:pressed {
    background-color: rgb(255, 163, 87);
    border: 1px solid #555;  /* 按下时改变边框颜色 */
	border-radius: 10px;
	font-size: 12px;  /* 鼠标按下时，字体缩小 */
}</string>
   </property>
   <property name="text">
    <string>退票</string>
   </property>
  </widget>
  <widget class="QLabel" name="airlineLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>100</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>航空公司|航班号</string>
   </property>
  </widget>
  <widget class="QLabel" name="price">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>30</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#price{
	color: rgb(255, 0, 0);
	font-size: 15px;
	font-weight: bold
}</string>
   </property>
   <property name="text">
    <string>价格</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>100</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>乘客</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
